content/includes/modal.html

```diff
@@ -0,0 +1,64 @@
+{% comment %}
+This modal is used on /education/apply-for-education-benefits/
+and should be extended as needed around the non-react vets.gov
+content.
+{% endcomment %}
+<section class="va-modal va-modal-split" style="display:none;" id="va-modal" role="alertdialog" tabIndex="-1">
+  <div class="va-modal-inner">
+    <h3 class="va-modal-title">Which form do you need to use?</h3>
+
+    <div class="va-modal-body rx-modal-body">
+      <div class="va-modal-split-container">
+        <div class="va-modal-split-left">
+          <h5>Servicemembers and Veterans applying for education benefits</h5>
+        </div>
+        <div class="va-modal-split-divider"></div>
+        <div class="va-modal-split-right">
+          <span>If you are:</span>
+          <ul>
+            <li>A student who wants to <b>change programs or schools</b></li>
+            <li>A spouse or dependent</li>
+            <li>A survivor of a Veteran</li>
+            <li>A candidate for <b>National Call to Service</b></li>
+          </ul>
+        </div>
+      </div>
+      <div class="va-modal-split-container">
+        <div class="va-modal-split-left">
+          <a id="va-modal-close" href="javascript:void(0)" class="usa-button-primary usa-button-big">Apply with Form 22-1990</a>
+        </div>
+        <div class="va-modal-split-divider"></div>
+        <div class="va-modal-split-right">
+          <a href="https://www.ebenefits.va.gov/" class="usa-button-primary usa-button-big usa-button-outline">Apply on eBenefits <span class="exit-icon">&nbsp;</span></a> 
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<script>
+(function($){
```

Comment:
In the React Modal there's some focus handling that would probably be useful here. I added it to help screenreaders navigate easier.